compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/declaration/FirExposedVisibilityDeclarationChecker.kt

```diff
@@ -190,7 +190,7 @@ object FirExposedVisibilityDeclarationChecker : FirMemberDeclarationChecker() {
         }
     }
 
-    private fun ConeKotlinType.findVisibilityExposure(
+    internal fun ConeKotlinType.findVisibilityExposure(
```

Comment:
I'm not sure why other utils (this one and below) need to be `internal` too?

Reply:
😛 Previously I moved checkTypeAlias there. But then forgot to change these back.